CHANGELOG.rst

```diff
@@ -2,10 +2,56 @@
 Changelog
 =========
 
+Version 16.0
+=============
+
+* Added support for ``qiskit == 1.2`` and ``qiskit-aer == 0.15``.
+* Dropped support for ``qiskit < 0.45``.
+* Updated the documentation.
+* :meth:`.IQMBackendBase.qubit_name_to_index` and :meth:`.IQMBackendBase.index_to_qubit_name` now
+  raise an error when using an invalid qubit name or index, rather than returning None.
+* Refactored :meth:`.IQMBackend.create_run_request` to improve user experience when using IQM
+  specific run options.
+* Introduced :attr:`.IQMBackendBase.physical_target` and :attr:`.IQMBackendBase.fake_target` to
+  represent the physical quantum architectures and a Qiskit-compatible version, respectively.
+* Moved the circuit serialization logic from :class:`.IQMProvider` to :mod:`iqm.qiskit_iqm.qiskit_to_iqm`.
+* Using the Qiskit transpiler with :class:`.IQMBackend`:
+
+  * You can now use the native Qiskit :func:`transpile` function to transpile a circuit to the IQM
+    Star architecture as long as your initial circuit does not use any resonators.
+  * The Qiskit transpiler now automatically uses the :class:`.IQMOptimizeSingleQubitGates` pass to
+    optimize single-qubit gates if ``optimization_level > 0``.
+  * There are many new transpiler plugins available that you can use as the ``scheduling_method``
+    argument in Qiskit's :func:`transpile` function. You can find them in the
+    `Qiskit documentation <https://docs.quantum.ibm.com/guides/transpiler-plugins>`_.
+  * If your circuit contains resonators, and optionally :class:`.MoveGate` operations, you can use
+    the :func:`.transpile_to_IQM` function to transpile your circuit for the IQM Star architecture.
+  * :func:`.transpile_to_IQM` can now restrict itself to use a subset of the qubits by specifying
+    the ``restrict_to_qubits`` argument. You will need to additionally provide a qubit mapping to the
+    :meth:`.IQMBackend.run` method to ensure that the correct qubits are used.
```

Comment:
Do we still need the qubit mapping in Qiskit, or would it be possible to get rid of it?

Reply:
This has to do with that the backend does not know you restricted to a certain set of qubits and the transpiler does not know this either, so the generated layout is BS. I cannot replace the layout of the transpiled circuit because there is no setter and I have been unable to find the pass that would apply a given layout to a circuit. There are only passes for creating a new layout to add to the circuit. Not to mention that the serialize_circuit function does not use the circuit layout to construct the IQM Circuit.